queries.sql

```diff
@@ -6,3 +6,25 @@ SELECT (name,escape_attempts) FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered is true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+BEGIN; UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is NULL;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Dear @odaialazzeh, to verify that the change was made and rollbacked we need to execute a select statement before and after the rollback statement. Please add `SELECT species from animals` before and after a `rollback` statement.
- [ ] Another thing is, we need to have a query Inside a transaction to delete all records in the animal's table, then roll back the transaction.
- [ ] Also, this statement has `ROLLBACK;` I cannot find your `BEGIN;` statement. Please make sure to have a visible beginning and end for your transactions alongside the select statements.